### DISCUSSION TITLE
120hz not working on battery saver

### DISCUSSION THREAD

**Discussion by DanielDEVp** [tribalfs/GalaxyMaxHzPub#241] [Q&A]
Hi, Iñve installed Galaxy Maxhz on my s21+(one ui 6.1) and I can't use 120hz on battery save mode.
![Screenshot_20241029_201350_Galaxy MaxHz](https://github.com/user-attachments/assets/f3b4afc0-a3e8-464f-9d15-8a6d70c9dbc1)

**Comment by tribalfs** [comment]
Ensure to read app description. You can read it here also:

![image](https://github.com/user-attachments/assets/72183c33-8dc3-4d2d-a291-d5c68954270b)
